changelog.d/9739.bugfix

```diff
@@ -0,0 +1 @@
+Fix #8518 and #3880.
```

Comment:
These won't get auto-linked by towncrier. Please spell out the bugs you are fixing - linking back to the GH issue number isn't normally necessary because anyone who is really interested can follow the link to the PR and thence to the GH issues.